### PR TITLE
Add default delivered duties to countries and region

### PR DESCRIPTION
**Pull request by paololim**
See corresponding API changes in: https://github.com/flowcommerce/api/pull/655

### CODE REVIEW

File: cleanse/cleanse.go [hunk 1]
```diff
@@ -228,6 +233,23 @@ func Cleanse() {
 	currencies := readCurrencies("data/original/currencies.json")
 	writeJson("data/cleansed/currencies.json", currencies)
 
+	writeJson("data/cleansed/country-duties.json",
+		toObjects(readCsv("data/original/country-duties.csv"),
+			func(record map[string]string) bool {
+				return record["duty"] != "" && record["duty"] != "--"
```
Comment (mbryzek) [inline cleanse/cleanse.go]: can remove ` && record["duty"] != "--"`

File: cleanse/cleanse.go [hunk 2]
```diff
@@ -228,6 +233,23 @@ func Cleanse() {
...
+			},
+			func(record map[string]string) interface{} {
+				return CountryDuty{
+					CountryCode:   record["country"],
+					DeliveredDuty: record["duty"],
+				}
+			},
+			func(record map[string]string) string {
+				return record["duty"] + record["country"]
```
Comment (mbryzek) [inline cleanse/cleanse.go]: this key is used to sort the data - so may be bit nicer to sort by country first: 
`record["country"] + record["duty"]`

File: final/final.go [hunk 3]
```diff
@@ -415,13 +427,14 @@ func eurozone(countries []common.Country) common.Region {
...
-		MeasurementSystems: measurementSystemsForCountries(theseCountries),
-		Timezones:          timezonesForCountries(theseCountries),
+		Id:                   "eurozone",
+		Name:                 "Eurozone",
+		Countries:            countryCodes,
+		Currencies:           currenciesForCountries(theseCountries),
+		Languages:            languagesForCountries(theseCountries),
+		MeasurementSystems:   measurementSystemsForCountries(theseCountries),
+		Timezones:            timezonesForCountries(theseCountries),
+		DefaultDeliveredDuty: theseCountries[0].DefaultDeliveredDuty,
```
Comment (mbryzek) [inline final/final.go]: not sure we need default delivered duty on region... in our use case each experience will have a country assigned to it that is the default -think we should just use that one.

File: cleanse/cleanse.go [hunk 4]
```diff
@@ -529,6 +550,13 @@ func toObjects(records []map[string]string, accepts acceptsFunction, f convertFu
 	return sortObjects(added)
 }
 
+func LoadCountryDuties() []CountryDuty {
```
Comment (mbryzek) [inline cleanse/cleanse.go]: This is fine - but given we will have one default per country, can also return this as a `map[string]CountryDuty` which will make lookup simpler

File: final/final.go (hunk 3 above)
Comment (paololim) [inline final/final.go]: https://github.com/flowcommerce/api/pull/656

**Pull request merged**